fleet-management-service/src/test/java/cz/fi/muni/pa165/service/BeanMappingServiceImplTest.java

```diff
@@ -0,0 +1,97 @@
+package cz.fi.muni.pa165.service;
+
+import cz.fi.muni.pa165.config.ServiceConfiguration;
+import cz.fi.muni.pa165.dto.*;
+import cz.fi.muni.pa165.entity.*;
+import cz.fi.muni.pa165.enums.Role;
+import cz.fi.muni.pa165.service.interfaces.BeanMappingService;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import javax.inject.Inject;
+import java.time.Year;
+import java.util.Date;
+
+/**
+ * @author Jozef Krcho
+ */
+@ContextConfiguration(classes = ServiceConfiguration.class)
+public class BeanMappingServiceImplTest extends AbstractTestNGSpringContextTests {
+
+    @Inject
+    private BeanMappingService beanMappingService;
+
+    @Test
+    public void testEmployeeMapping() {
```

Comment:
They don't have test like this, I think this is allright ... If I don't do this thes. We wouldn't find that Vehicle java.time.Year can't be mapped ... we need different field type.